code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -46,6 +46,12 @@
 			return spec_return
 
 	if(mind)
+		if (mind.martial_art && mind.martial_art.dodge_chance)
+			if(!lying && dna && !dna.check_mutation(HULK))
+				if(mind.martial_art.dodge_chance >= rand(0,100))

```

Comment:
just use prob(mind.martial_art.dodge_chance)

Reply:
and i know this is the second time i've done this but consistency in the rng procs is important to me

Reply:
this made me giggle alrighty